closed/adds/jdk/src/share/native/jdk/crypto/jniprovider/NativeCrypto.c

```diff
@@ -255,16 +260,16 @@ JNIEXPORT void JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_CBCInit
             break;
     }
 
-    ivNative  = (unsigned char*)((*env)->GetByteArrayElements(env, iv,  &isCopy));
-    keyNative = (unsigned char*)((*env)->GetByteArrayElements(env, key, &isCopy));
+    ivNative = (unsigned char*)((*env)->GetByteArrayElements(env, iv, 0));
+    keyNative = (unsigned char*)((*env)->GetByteArrayElements(env, key, 0));
```

Comment:
These calls should also be error checked.

Reply:
Done.. added null checks.